__tests__/unit/old-structure/BeagleUIView.spec.ts

```diff
@@ -94,9 +92,7 @@ describe('BeagleUIView', () => {
     await view.getNavigator().pushView({ url: path })
     // @ts-ignore
     const expectedErrors = [
-      new BeagleExpiredCacheError(url),
-      new BeagleNetworkError(url, createHttpResponse()),
-      new BeagleCacheError(url),
+      new BeagleNetworkError(url, createHttpResponse(),500,'GET'),
```

Comment:
```suggestion
      new BeagleNetworkError(url, createHttpResponse(), 500, 'GET'),
```
It would be nice to add the space after the "," to keep a pattern.